packages/launcher.rb

```diff
@@ -0,0 +1,27 @@
+require 'package'
+
+class Launcher < Package
+  description 'Add GUI applications shortcut to Chrome/Chromium OS launcher'
+  homepage 'https://github.com/supechicken/chromebrew/tree/master/launcher'
+  version '5.0'
+  source_url 'https://github.com/supechicken/chromebrew/raw/master/launcher/launcher.rb'
+  source_sha256 '117615ee5f2cf5096c8fef274cfa3cc94c2161b5b23e080ff199081c18b5c0a7'
+  
+  depends_on 'libyaml'
+  depends_on 'imagemagick7'
+
+  def self.install
+    system('gem', 'install', 'em-websocket')
```

Comment:
Can we include the gem dependency files in the package?  I'm thinking this could be done similar to how we add perl or python dependencies in other packages.  It might even make sense to package the `em-websocket` gem in a separate package and add that as a dependency here.